README.md

```diff
@@ -0,0 +1,38 @@
+# Wemos Websocket LED
+
+The Arduino's Wemos Websocket project, by Clever Cloud.
+
+The Wemos is connected to a LED strip. This strip is a gradient defined by two colors. When a new color is selected, the gradient goes from the new color to the old one.
+
+## Interact with the strip
+
+You can change the color with the interface given by the [wemos-ws-led-backend](https://github.com/CleverCloud/wemos-ws-led-backend) project or with a simple request :
```

Comment:
This is a bit misleading (the "simple request" goes to the backend so this sentence is a bit weird)

By the way, there is no space before any punctuation in English.